Makefile

```diff
@@ -120,9 +120,9 @@ run-apiserver:
 
 run-build:
 	GOGC=1000 go install -v \
-		-ldflags '-X main.version=$(if $(BUILD_VERSION),$(BUILD_VERSION),devbuild)' \
-		./volcli/volcli/ ./volplugin/volplugin/ ./apiserver/apiserver/ ./volsupervisor/volsupervisor/ ./volmigrate/volmigrate/
-	cp $(GUESTBINPATH)/* bin/
+		 -ldflags '-X main.version=$(if ${BUILD_VERSION},${BUILD_VERSION},devbuild)' \
```

Comment:
Is your editor adding the extra spaces at the beginning of the lines here?  I removed them in the commit which added volmigrate